### PR TITLE
Added appEpics

### PR DESCRIPTION
**Pull request by kappu72**
This pull lets pass an object with app specific Epics
If Epic's name, in appEpics, is the same the one in plugins this will override plugin's one

### CODE REVIEW

File: web/client/stores/StandardStore.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const {createEpicMiddleware} = require('redux-observable');
 const SecurityUtils = require('../utils/SecurityUtils');
 const ListenerEnhancer = require('@carnesen/redux-add-action-listener-enhancer').default;
 
-module.exports = (initialState = {defaultState: {}, mobile: {}}, appReducers = {}, plugins, storeOpts) => {
+module.exports = (initialState = {defaultState: {}, mobile: {}}, appReducers = {}, appEpics = {}, plugins, storeOpts) => {
```
Comment (mbarto) [inline web/client/stores/StandardStore.js]: New arguments at the end, for compatibility reasons, please

File: web/client/utils/__tests__/PluginUtils-test.js [hunk 2]
```diff
@@ -72,5 +73,11 @@ describe('PluginsUtils', () => {
         expect(desc1.items[0].cfg).toExist();
 
     });
+    it('combineEpics', () => {
+        const plugins = {MapSearchPlugin: MapSearchPlugin};
+        const appEpics = {appEpics: (actions$) => actions$};
```
Comment (offtherailz) [inline web/client/utils/__tests__/PluginUtils-test.js]: This sounds like an infinite loop. Is not executed but it should. Please remove it.

File: web/client/stores/StandardStore.js (hunk 1 above)
Comment (offtherailz) [inline web/client/stores/StandardStore.js]: This signature is better. This doesn't break the binding with StandardApp or other apps.

Comment (kappu72) [inline web/client/stores/StandardStore.js]: Binding appEpics wouldn't be possible

**Pull request merged**